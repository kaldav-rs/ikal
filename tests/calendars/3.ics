BEGIN:VCALENDAR
VERSION:2.0
PRODID:+//IDN bitfire.at//DAVdroid/1.0.8 ical4android ical4j/2.x
BEGIN:VTODO
DTSTAMP:20160515T164310Z
UID:10442264-14b1-47cf-9263-69447c0e0d7f
CREATED:20160515T094801Z
LAST-MODIFIED:20160515T094801Z
SUMMARY:the science of good cooking +book
LOCATION:
STATUS:COMPLETED
PERCENT-COMPLETE:100
COMPLETED:20160515T192354
END:VTODO
END:VCALENDAR
