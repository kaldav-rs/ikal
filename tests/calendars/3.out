Ok(
    VCalendar {
        prodid: "+//IDN bitfire.at//DAVdroid/1.0.8 ical4android ical4j/2.x",
        version: "2.0",
        calscale: None,
        method: None,
        events: [],
        freebusy: [],
        journals: [],
        todo: [
            VTodo {
                dtstamp: 2016-05-15T18:43:10+02:00,
                uid: "10442264-14b1-47cf-9263-69447c0e0d7f",
                class: None,
                completed: Some(
                    2016-05-15T19:23:54+02:00,
                ),
                created: Some(
                    2016-05-15T11:48:01+02:00,
                ),
                dtstart: None,
                geo: None,
                last_modified: Some(
                    2016-05-15T11:48:01+02:00,
                ),
                location: Some(
                    "",
                ),
                organizer: None,
                percent_complete: Some(
                    100,
                ),
                priority: None,
                recurid: None,
                sequence: None,
                status: Some(
                    Completed,
                ),
                summary: Some(
                    "the science of good cooking +book",
                ),
                url: None,
                rrule: None,
                due: None,
                duration: None,
                attach: [],
                attendee: [],
                categories: [],
                comment: [],
                contact: [],
                exdate: [],
                rstatus: [],
                related_to: [],
                resources: [],
                rdate: [],
                x_prop: {},
                iana_prop: {},
            },
        ],
        timezones: [],
        x_prop: {},
        iana_prop: {},
    },
)
